Dockerfile.php7

```diff
@@ -0,0 +1,7 @@
+FROM php:7.0
```

Comment:
Can you just update the main Dockerfile? The default one should probably be updated to PHP7, there's no reason to maintain one for PHP5 anymore.